.github/agents/performance.agent.md

```
@@ -0,0 +1,63 @@
+
+## Optimization Rules
+
+- Always measure before and after
+- Optimize the biggest bottleneck first (Amdahl's law)
+- Don't optimize what isn't slow (premature optimization)
+- Cache invalidation must be correct (stale data is a bug)
+- Set resource limits (memory, CPU, connections, timeouts)
+
+## Output Format
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)